models/batch.py

```
@@ -0,0 +1,5 @@
+from itertools import combinations
```

Ruff (0.8.2)

`itertools.combinations` imported but unused

Remove unused import: `itertools.combinations`

(F401)